packages/styles/stories/introduction.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Styles/About the Style injection system" />
+
+# About the Salt Theme
+
+## Component style injection
+
+In order to support uses of Salt that utilize multiple external windows such as desktop applications. We offer two packages that contain hooks to facilitate this: `@salt-ds/window` and `@salt-ds/styles`.
+`@salt-ds/styles` contains the `useComponentCssInjection` hook while `@salt-ds/window` contains the `useWindow` hook.
+
+Below is an example component illustrating some important concepts:
+
+```tsx
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import exampleCss from "./Example.css";
+
+const Example = (props) => {
+  const { window: targetWindow } = useWindow();
+  useComponentCssInjection({
+    id: "salt-example",
+    css: exampleCss,
+    window: targetWindow,
+  });
+
+  return <svg className="saltExample-svg">{props.children}</svg>;
+};
+```
+
+- All Salt component CSS must be imported using the default export, the imported CSS should be injected by calling the `useComponentCssInjection` hook.
+- The current window the component is being rendered into is retrieved using the `useWindow` hook.
+- Only one component should be stored per typescript jsx file, however a component may import and inject multiple css files.
```

Comment:
not sure what exactly are we talking about here. Does that mean the system will break if having multiple components in a single file?